### PR TITLE
Add a demo creation script

### PR DESCRIPTION
**Pull request by fedepaol**
We setup an external frr container and provide multiple configurations to showcase frr-k8s features.

### CODE REVIEW

File: hack/demo/demo.sh
Comment (oribon) [inline hack/demo/demo.sh:1]: iiuc this relies on first creating a kind cluster with everything? could/should this be smoother by first deploying from the makefile target?

File: hack/demo/configs/templates/advertise.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+apiVersion: frrk8s.metallb.io/v1beta1
+kind: FRRConfiguration
+metadata:
+  name: advsome
```
Comment (oribon) [inline hack/demo/configs/templates/advertise.yaml:4]: nit: rename these to something clearer - advertise-filtered, advertise-all, nodeselectors, ..

File: hack/demo/demo.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+for i in configs/*.yaml; do
+	rm $i
+done
+
+cp configs/templates/*.yaml configs/
+
+for i in configs/*.yaml; do
+    sed -i "s/NEIGHBOR_IP/$FRR_IP/g" $i
+done
```
Comment (oribon) [inline hack/demo/demo.sh:30]: can we also echo here that the setup is complete, yamls are in `hack/demo/..` ?

File: hack/demo/demo.sh
Comment (fedepaol) [reply hack/demo/demo.sh:1]: done

File: hack/demo/configs/templates/advertise.yaml (hunk 1 above)
Comment (fedepaol) [reply hack/demo/configs/templates/advertise.yaml]: done

File: hack/demo/demo.sh (hunk 2 above)
Comment (fedepaol) [reply hack/demo/demo.sh:30]: done

File: hack/demo/frr/frr.conf
Comment (oribon) [inline hack/demo/frr/frr.conf:1]: nit: I think this can be .gitignored ?

**Review by oribon — COMMENTED** [review]
other than that lgtm

**Pull request merged**